migrations.md

```diff
@@ -0,0 +1,11 @@
+If a contributor wants to change the schema, they must create a migration. Current migrations have been created with the following commands:
```

Comment:
We deleted it but it must have come back during a rebase without our realizing it. When we push up again it'll be gone.